extensions/gmsa-dc/v1/Setup-gMSA.ps1

```diff
@@ -276,3 +289,5 @@ if (Get-AdGroupMember -Identity "Enterprise Admins" | Select-String -Pattern "gm
     Restart-Computer -Force
 }
 
+# Make sure a folder is created on Active Directory server for file reading/writing test
+do {$Fileresult = CreateRemoteFolder} while ($Fileresult -eq $false)
```

Comment:
this should be added if statement.  This file gets runs twice, the first one [adds the domain admin user to the Admin group then restarts](https://github.com/kubernetes-sigs/windows-testing/blob/f830ab4d04aa01e8ce9e8d68b1471ffb35a79c7e/extensions/gmsa-dc/v1/Setup-gMSA.ps1#L270-L276) and the [second run does the setup](https://github.com/kubernetes-sigs/windows-testing/blob/f830ab4d04aa01e8ce9e8d68b1471ffb35a79c7e/extensions/gmsa-dc/v1/Setup-gMSA.ps1#L239-L269).  This should run in the second iteration after the gMSA account is created https://github.com/kubernetes-sigs/windows-testing/blob/f830ab4d04aa01e8ce9e8d68b1471ffb35a79c7e/extensions/gmsa-dc/v1/Setup-gMSA.ps1#L257

Reply:
Got it, thanks James!